image/mkosi/mkosi.skeleton/etc/profile.d/constellation.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+export PATH=/run/state/bin:${PATH}
+export KUBECONFIG=/etc/kubernetes/admin.conf
+alias k=kubectl
```

Comment:
```suggestion
```
Nit but I wouldn't define this on the read-only disk. Maymay maybe only for debug images.

Reply:
Why would you not define aliases on read-only disks? I don't see how this would interfere with anything. It just makes debugging a lot easier in my opinion.

Reply:
It's not interfering, but it's more a general UX argument for me that:

- If I like the alias, I can set it myself easily when debugging stuff in the shell  (if it's not defined on the read-only disk)
- If I don't like the alias, I have to unset this manually every single time (if it's defined on the read-only disk)

I don't have a strong opinion on this, since it's just an alias after all. But personally I prefer keeping the environment as clean as is from personal preferences (even if I left a ❤️ for nano as default editor above previously :P)